### ISSUE TITLE
[1.20.1] Server crashing

### ISSUE THREAD

**Issue by Sleepyhalo1** [TheRealZeher/DimensionalPockets-II#113]
When i launch my server it gives me this in the crash report
[Uploading crash-2024-04-10_14.59.53-server.txt…]()

**Comment by TheCosmicNebula** [comment]
Hi, thanks for the report.

That link doesnt provide me with the crash report? Are you able to re-upload it please?

**Comment by Sleepyhalo1** [comment]
```
----Minecraft Crash Report ----
// My bad.

Time: 2024-04-10 15:48:37
Description: Ticking block entity

java.lang.NullPointerException: Cannot invoke "com.tcn.dimensionalpocketsii.pocket.core.gson.PocketChunkInfo.getDominantChunk()" because "this.chunk_info" is null
	at com.tcn.dimensionalpocketsii.pocket.core.Pocket.getDominantChunkPos(Pocket.java:407) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at com.tcn.dimensionalpocketsii.pocket.core.Pocket.updateBaseConnectors(Pocket.java:1129) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at com.tcn.dimensionalpocketsii.pocket.core.block.entity.BlockEntityModuleConnector.sendUpdates(BlockEntityModuleConnector.java:120) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at com.tcn.dimensionalpocketsii.pocket.core.block.entity.BlockEntityModuleConnector.tick(BlockEntityModuleConnector.java:367) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.m_142224_(LevelChunk.java:689) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.m_142224_(LevelChunk.java:782) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:classloading}
	at net.minecraft.world.level.Level.m_46463_(Level.java:468) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:aether.mixins.json:common.accessor.LevelAccessor,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:351) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:aether.mixins.json:common.accessor.ServerLevelAccessor,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:893) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:283) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Suspected Mod: 
	Dimensional Pockets II (dimensionalpocketsii), Version: 9.0.8.0
		Issue tracker URL: https://github.com/TheCosmicNebula/DimensionalPocketsII/issues
		at TRANSFORMER/dimensionalpocketsii@9.0.8.0/com.tcn.dimensionalpocketsii.pocket.core.Pocket.getDominantChunkPos(Pocket.java:407)
Stacktrace:
	at com.tcn.dimensionalpocketsii.pocket.core.Pocket.getDominantChunkPos(Pocket.java:407) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at com.tcn.dimensionalpocketsii.pocket.core.Pocket.updateBaseConnectors(Pocket.java:1129) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at com.tcn.dimensionalpocketsii.pocket.core.block.entity.BlockEntityModuleConnector.sendUpdates(BlockEntityModuleConnector.java:120) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at com.tcn.dimensionalpocketsii.pocket.core.block.entity.BlockEntityModuleConnector.tick(BlockEntityModuleConnector.java:367) ~[dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar%23122!/:1.20.1-9.0.8.0-universal] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.m_142224_(LevelChunk.java:689) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.m_142224_(LevelChunk.java:782) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:classloading}
	at net.minecraft.world.level.Level.m_46463_(Level.java:468) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:aether.mixins.json:common.accessor.LevelAccessor,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:351) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:aether.mixins.json:common.accessor.ServerLevelAccessor,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:A}
-- Block entity being ticked --
Details:
	Name: dimensionalpocketsii:tile_entity_connector // com.tcn.dimensionalpocketsii.pocket.core.block.entity.BlockEntityModuleConnector
	Block: Block{dimensionalpocketsii:block_wall_connector}[mode=0,type=0]
	Block location: World: (0,0,-1216), Section: (at 0,0,0 in 0,0,-76; chunk contains blocks 0,0,-1216 to 15,255,-1201), Region: (0,-3; contains chunks 0,-96 to 31,-65, blocks 0,0,-1536 to 511,255,-1025)
	Block: Block{dimensionalpocketsii:block_wall_connector}[mode=0,type=0]
	Block location: World: (0,0,-1216), Section: (at 0,0,0 in 0,0,-76; chunk contains blocks 0,0,-1216 to 15,255,-1201), Region: (0,-3; contains chunks 0,-96 to 31,-65, blocks 0,0,-1536 to 511,255,-1025)
Stacktrace:
	at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.m_142224_(LevelChunk.java:689) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.m_142224_(LevelChunk.java:782) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:classloading}
	at net.minecraft.world.level.Level.m_46463_(Level.java:468) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:aether.mixins.json:common.accessor.LevelAccessor,pl:mixin:A}
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:351) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:aether.mixins.json:common.accessor.ServerLevelAccessor,pl:mixin:APP:create.mixins.json:accessor.ServerLevelAccessor,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:893) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:283) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {}


-- Affected level --
Details:
	All players: 0 total; []
	Chunk stats: 2755
	Level dimension: dimensionalpocketsii:pocket
	Derived: true
	Level spawn location: World: (0,66,0), Section: (at 0,2,0 in 0,4,0; chunk contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 10769186 game time, 11377511 day time
	Level name: world
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Level weather: Rain time: 86610 (now: false), thunder time: 68496 (now: false)
	Known server brands: forge
	Removed feature flags: 
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
Stacktrace:
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:893) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:283) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23129!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dankstorage.mixins.json:MinecraftServerAccess,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {}


-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 17.0.7, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 4885181840 bytes (4658 MiB) / 6987710464 bytes (6664 MiB) up to 12884901888 bytes (12288 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: 11th Gen Intel(R) Core(TM) i7-11370H @ 3.30GHz
	Identifier: Intel64 Family 6 Model 140 Stepping 1
	Microarchitecture: Tiger Lake
	Frequency (GHz): 3.30
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: Intel(R) Iris(R) Xe Graphics
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 1024.00
	Graphics card #0 deviceId: 0x9a49
	Graphics card #0 versionInfo: DriverVersion=30.0.101.1631
	Graphics card #1 name: NVIDIA GeForce RTX 3050 Ti Laptop GPU
	Graphics card #1 vendor: NVIDIA (0x10de)
	Graphics card #1 VRAM (MB): 4095.00
	Graphics card #1 deviceId: 0x25a0
	Graphics card #1 versionInfo: DriverVersion=31.0.15.1659
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 16384.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 35019.33
	Virtual memory used (MB): 18154.14
	Swap memory total (MB): 10752.00
	Swap memory used (MB): 0.00
	JVM Flags: 1 total; -Xmx12G
	Server Running: true
	Player Count: 0 / 20; []
	Data Packs: vanilla, mod:mixinextras (incompatible), mod:cumulus_menus, mod:aether, mod:spectrelib (incompatible), mod:inventorypets (incompatible), mod:charmofundying (incompatible), mod:nitrogen_internals, mod:forge, mod:toms_storage (incompatible), mod:curios (incompatible), mod:cosmeticarmorreworked, mod:dimensionalpocketsii, mod:dankstorage (incompatible), mod:cosmoslibrary, builtin/aether_accessories, mod:toolbelt (incompatible), mod:journeymap (incompatible), mod:fishingupgrades, mod:flywheel, mod:create, mod:createaddition (incompatible)
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Experimental
	Is Modded: Definitely; Server brand changed to 'forge'
	Type: Dedicated Server (map_server.txt)
	ModLauncher: 10.0.9+10.0.9+main.dcd20f30
	ModLauncher launch target: forgeserver
	ModLauncher naming: srg
	ModLauncher services: 
		mixin-0.8.5.jar mixin PLUGINSERVICE 
		eventbus-6.0.5.jar eventbus PLUGINSERVICE 
		fmlloader-1.20.1-47.2.21.jar slf4jfixer PLUGINSERVICE 
		fmlloader-1.20.1-47.2.21.jar object_holder_definalize PLUGINSERVICE 
		fmlloader-1.20.1-47.2.21.jar runtime_enum_extender PLUGINSERVICE 
		fmlloader-1.20.1-47.2.21.jar capability_token_subclass PLUGINSERVICE 
		accesstransformers-8.0.4.jar accesstransformer PLUGINSERVICE 
		fmlloader-1.20.1-47.2.21.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-10.0.9.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-10.0.9.jar fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		server-1.20.1-20230612.114412-srg.jar             |Minecraft                     |minecraft                     |1.20.1              |DONE      |Manifest: NOSIGNATURE
		mixinextras-forge-0.2.0-beta.9.jar                |MixinExtras                   |mixinextras                   |0.2.0-beta.9        |DONE      |Manifest: NOSIGNATURE
		ToolBelt-1.20-1.20.0.jar                          |Tool Belt                     |toolbelt                      |1.20.0              |DONE      |Manifest: NOSIGNATURE
		cumulus_menus-1.20.1-1.0.0-neoforge.jar           |Cumulus                       |cumulus_menus                 |1.20.1-1.0.0-neoforg|DONE      |Manifest: NOSIGNATURE
		FishingUpgrades_1.0.2.jar                         |fishingupgrades               |fishingupgrades               |1.0.2               |DONE      |Manifest: NOSIGNATURE
		aether-1.20.1-1.3.1-neoforge.jar                  |The Aether                    |aether                        |1.20.1-1.3.1-neoforg|DONE      |Manifest: NOSIGNATURE
		spectrelib-forge-0.13.15+1.20.1.jar               |SpectreLib                    |spectrelib                    |0.13.15+1.20.1      |DONE      |Manifest: NOSIGNATURE
		inventorypets-1.20.1-2.1.1.jar                    |Inventory Pets                |inventorypets                 |2.1.1               |DONE      |Manifest: NOSIGNATURE
		charmofundying-forge-6.5.0+1.20.1.jar             |Charm of Undying              |charmofundying                |6.5.0+1.20.1        |DONE      |Manifest: NOSIGNATURE
		nitrogen_internals-1.20.1-1.0.7-neoforge.jar      |Nitrogen                      |nitrogen_internals            |1.20.1-1.0.7-neoforg|DONE      |Manifest: NOSIGNATURE
		forge-1.20.1-47.2.21-universal.jar                |Forge                         |forge                         |47.2.21             |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		toms_storage-1.20-1.6.6.jar                       |Tom's Simple Storage Mod      |toms_storage                  |1.6.6               |DONE      |Manifest: NOSIGNATURE
		curios-forge-5.7.2+1.20.1.jar                     |Curios API                    |curios                        |5.7.2+1.20.1        |DONE      |Manifest: NOSIGNATURE
		cosmeticarmorreworked-1.20.1-v1a.jar              |CosmeticArmorReworked         |cosmeticarmorreworked         |1.20.1-v1a          |DONE      |Manifest: 5e:ed:25:99:e4:44:14:c0:dd:89:c1:a9:4c:10:b5:0d:e4:b1:52:50:45:82:13:d8:d0:32:89:67:56:57:01:53
		flywheel-forge-1.20.1-0.6.10-7.jar                |Flywheel                      |flywheel                      |0.6.10-7            |DONE      |Manifest: NOSIGNATURE
		create-1.20.1-0.5.1.f.jar                         |Create                        |create                        |0.5.1.f             |DONE      |Manifest: NOSIGNATURE
		journeymap-1.20.1-5.9.20-forge.jar                |Journeymap                    |journeymap                    |5.9.20              |DONE      |Manifest: NOSIGNATURE
		dimensionalpocketsii-1.20.1-9.0.8.0-universal.jar |Dimensional Pockets II        |dimensionalpocketsii          |9.0.8.0             |DONE      |Manifest: NOSIGNATURE
		dankstorage-forge-1.20.1-8.jar                    |Dank Storage                  |dankstorage                   |8                   |DONE      |Manifest: NOSIGNATURE
		createaddition-1.20.1-1.2.3.jar                   |Create Crafts & Additions     |createaddition                |1.20.1-1.2.3        |DONE      |Manifest: NOSIGNATURE
		cosmos-library-1.20.1-10.5.1.0-universal.jar      |Cosmos Library                |cosmoslibrary                 |10.5.1.0            |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: 12938e47-9d45-43d4-94c5-457315450e12
	FML: 47.2
	Forge: net.minecraftforge:47.2.21
```

**Comment by Sleepyhalo1** [comment]
it wouldnt let me send it prop so i just did it like this sry

**Comment by TheCosmicNebula** [comment]
Closed out - will be investigated in the 1.21 NeoForge update.
